packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -248,11 +248,12 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     maxDominantSpeakers: MAX_AUDIO_DOMINANT_SPEAKERS
   });
 
-  /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
-  const localCameraCycleButton = (localVideoCameraCycleButtonProps): JSX.Element => {
-    const ariaDescription = _formatString(strings.localVideoSelectedDescription, {
-      cameraName: localVideoCameraCycleButtonProps.selectedCamera.name
-    });
+  const LocalCameraCycleButton = useCallback((): JSX.Element => {
```

Comment:
I would discourage using `CapitalCase` name for a local variable, even if it behaves like a `React.Component`.

Two options:
* Extract into an actual component below in this file and use that.
* Name `camelCase` like any other variable.

Reply:
I had it extracted as a component but made the changelist more confusing to review -- hard to spot that just a useCallback is now added. I have a follow up PR that will be moving this (and the whole local video tile) to its own file so will be cleaned up shortly 👍